### ISSUE TITLE
Notes are not being persisted after local environment shut down

### ISSUE THREAD

**Issue by jeet-parekh** [standardnotes/web#455]
I discovered standardnotes just last week. It's awesome! Thank you for your efforts.

I have been testing SN by running it locally. I tried both the Docker image and the manual build. Both development and production mode. I have been able to get standardnotes up and running each time.

I discovered that if I shut down the server and bring it back up, it loses my notes. More specifically, after shutting down the server, if I close my browser, then it loses my notes. If I shut down the server and bring it back up without closing my browser in between, then my notes are not lost.

I'm not sure if this is a bug or intended behaviour.

**Comment by mobitar** [comment]
Sounds like you may be using either an incognito tab in your browser, or your browser's settings are set to delete site data after it closes. Try with another browser and see if you get the same results.

**Comment by jeet-parekh** [comment]
You're right. My browser clears the site data after it closes. I'm not using an incognito tab though.
I assumed SN saves the notes to disk while shutting down. Is that not the case?

**Comment by mobitar** [comment]
When using the web app we can't access disk. Only whatever data storing mechanisms the browser allows us (IndexedDB in this case). You can use the desktop app though if you want on-disk persistence.

**Comment by jeet-parekh** [comment]
I see. I assumed it would write to disk because of the ruby code in there. I have zero ruby knowledge though. I'll be using the desktop app now.
Just curious, what does the ruby code do?

**Comment by jeet-parekh** [comment]
I experimented with SN docker quite a lot. Found a couple of oddities and possible enhancements. I'll open another issue for that after a while.
